### DISCUSSION TITLE
meta field in client table

### DISCUSSION THREAD

**Discussion by sagarshah1983** [ory/hydra#2244] [Q&A] [answered]
How's the meta column being referred in the client table?
Are there any predefined properties that can be applied at per client level here? Or can it be used for any custom properties as per implementation need as well?
Any suggestions are highly appreciated!

**Comment by aeneasr** [comment, marked as answer]
The field can be set to whatever you want it to

**Reply by sagarshah1983** [reply]
Thank you for clarifying @aeneasr . That helps!
